src/pybind/mgr/dashboard/exceptions.py

```diff
@@ -45,6 +45,14 @@ def code(self):
         return str(abs(self.errno)) if self.errno is not None else 'Error'
 
 
+# Subclass of the DashboardException
```

Comment:
This comment seems redundant as the class definition is very clear:
```class InvalidCredentialsError(DashboardException)```

Reply:
Fair enough. I removed the comment  :)